src/libraries/Native/build-native.cmd

```diff
@@ -97,7 +97,12 @@ goto :SetupDirs
 echo Commencing build of native components
 echo.
 
-if /i "%__BuildArch%" == "wasm" set __sourceDir=%~dp0\Unix
+set __ExtraCmakeParams="-DCMAKE_REPO_ROOT=%__cmakeRepoRoot%"
+
+if /i "%__BuildArch%" == "wasm" (
+    set __sourceDir=%~dp0\Unix
+    set __ExtraCmakeParams=%__ExtraCmakeParams% "-DCLR_ENG_NATIVE_DIR=%__engNativeDir%" "-DCMAKE_BUILD_TYPE=%CMAKE_BUILD_TYPE%"
```

Comment:
`CLR_ENG_NATIVE_DIR` comes from `eng\native\configurepaths.cmake` in other places. Can this be on the same plan?

Reply:
I think this was hanging around previously and thought it was necessary.  I saw that configurepaths.cmake was already included, so taking it out worked.